gigareview/common/models/EMReportJob.php

```diff
@@ -14,9 +17,50 @@ class EMReportJob extends \yii\base\BaseObject implements \yii\queue\JobInterfac
     public string $fetchDate ;
     public string $scope;
 
+
     public function execute($queue)
     {
-        // TODO: Implement execute() method.
+        if ($this->scope === "manuscripts" && $this->content !== "No Results") {
+            $tempManuscriptCsvFile = tempnam(sys_get_temp_dir(), "test-manuscripts").".csv";
+
+            file_put_contents($tempManuscriptCsvFile, $this->content);
```

Comment:
I would move these 2 lines into `parseReport(string $emReportPath)` to try to keep this `execute($queue)` function as clean as possible.